Jenkinsfile

```diff
@@ -266,13 +266,14 @@ pipeline {
           }
           steps {
             sh('export')
-            sh('make distclean')
+            sh('make px4_sitl_rtps')
```

Comment:
You should still run `make distclean` first. If a previous build fails for any reason it won't clean up after itself.

Reply:
Makes sense. Fixed.